tests/fixtures/api-statistics.py

```diff
@@ -10,8 +10,6 @@ def statistics():
             "populationId": "2d64abe2ca9df61348ed8z9b",
             "population": "Singlets",
             "uniquePopulationName": "Singlets",
-            "parentPopulationId": None,
-            "parentPopulation": "Ungated",
```

Comment:
ack sorry, the scope of #125 was a bit unclear. Stats still include these two props.

Reply:
:thu